src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -90,6 +91,13 @@ class ContainerBuilder extends Container implements TaggedContainerInterface
      */
     private $expressionLanguageProviders = array();
 
+    public function __construct(ParameterBagInterface $parameterBag = null)
+    {
+        parent::__construct($parameterBag);
+
+        $this->trackResources = interface_exists('Symfony\Component\Config\Resource\ResourceInterface');
```

Comment:
Could this be simply `interface_exists(ResourceInterface::class);`? The class is already imported anyway